### DISCUSSION TITLE
Web-ECharts 地图可视化 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#331] [Announcements]
# Web-ECharts 地图可视化 | Zi-Zi's Journey

踏万千河山 风雨飘 摇自由停歇

https://gz-metal-cell.github.io/2024/04/22/Web-ECharts%20%E5%9C%B0%E5%9B%BE%E5%8F%AF%E8%A7%86%E5%8C%96/

**Comment by promefire** [comment]
太好看了吧😭Metal神！😭

**Reply by GZ-Metal-Cell** [reply]
😭还没有应舍友的要求来到了山西😭那里一出车站就是晋城的旅游宣传图😭不然就更好看了😭

**Comment by GZ-Metal-Cell** [comment]
😭寄😭手机版的 edge 不显示贴图😭
